tox.ini

```diff
@@ -28,7 +28,7 @@ commands =
     python -V
     pandoc --version
     no_extras: python -m pip uninstall -q -y tomli
-    coverage run -p -m pytest -Wd {posargs}
+    coverage run -p -m pytest -Wd -c setup.cfg {posargs}
```

Comment:
This doesn't seem to be necessary.  Without it, pytest still reports that it used setup.cfg as the config file.

Reply:
As noted above, it's necessary because pytest will discover the new `pyproject.toml` file and attempt to import `tomli` on Pythons < 3.11 so it can check the TOML file for its config. [This causes the `py39-no_extras` tox environment to fail](https://github.com/nedbat/scriv/actions/runs/13786817250/job/38556677243#step:10:276) as pytest is starting.

Setting this explicitly circumvents pytest's config auto-detection and avoids its import of `tomli`.